Include/httpClient/config.h

```diff
@@ -57,8 +58,12 @@
     #else
         #define HC_DATAMODEL HC_DATAMODEL_ILP32
     #endif
-#elif defined(__ANDROID__)
-    #define HC_PLATFORM HC_PLATFORM_ANDROID
+#elif defined(__linux__)
+    #if defined(__ANDROID)
```

Comment:
shouldn't this be `__ANDROID__`?

Reply:
you're right I'll resolve this in my next push. Need to make the templates